articles/cosmos-db/request-units.md

```diff
@@ -50,14 +50,14 @@ We recommend getting started by watching the following video, where Aravind Rama
 > 
 
 ## Specifying request unit capacity in Azure Cosmos DB
-When starting a new collection, table or graph, you specify the number of request units per second (RU per second) you want reserved. Based on the provisioned throughput, Azure Cosmos DB allocates physical partitions to host your collection and splits/rebalances data across partitions as it grows.
+When starting a new container, table or graph, you specify the number of request units per second (RU per second) you want reserved. Based on the provisioned throughput, Azure Cosmos DB allocates physical partitions to host your container and splits/rebalances data across partitions as it grows.
```

Comment:
At the beginning of the sentence, please change "container, table or graph" to simply "container".